client/tokenized-payment-request/index.js

```diff
@@ -0,0 +1,79 @@
+/* global jQuery, wcpayPaymentRequestParams */
+/**
+ * External dependencies
+ */
+import { doAction } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import WCPayAPI from '../checkout/api';
+import PaymentRequestCartApi from './cart-api';
+import WooPaymentsPaymentRequest from './payment-request';
+import paymentRequestButtonUi from './button-ui';
+import './compatibility/wc-deposits';
+import './compatibility/wc-order-attribution';
+import './compatibility/wc-product-variations';
+
+import '../checkout/express-checkout-buttons.scss';
+
+jQuery( ( $ ) => {
+	// Don't load if blocks checkout is being loaded.
+	if (
+		wcpayPaymentRequestParams.has_block &&
+		wcpayPaymentRequestParams.button_context !== 'pay_for_order'
```

Comment:
```suggestion
import paymentRequestButtonUi from './button-ui';
import getPaymentRequestData from './frontend-utils';
import './compatibility/wc-deposits';
import './compatibility/wc-order-attribution';
import './compatibility/wc-product-variations';

import '../checkout/express-checkout-buttons.scss';

jQuery( ( $ ) => {
	// Don't load if blocks checkout is being loaded.
	if (
		getPaymentRequestData( 'has_block' ) &&
		getPaymentRequestData( 'button_context' ) !== 'pay_for_order'
```
No biggie, but just sayin'. 😅 Couple other places here where this could apply too, up to you.